openmdao/core/tests/test_connections_in_configure.py

```diff
@@ -244,13 +244,7 @@ def test_queue_subcomponent_io_in_configure(self):
 
         p.model.add_subsystem('g', subsys=g)
 
-        with self.assertRaises(RuntimeError) as e:
-            p.setup()
-
-        err_msg = str(e.exception)
-        expected = "Cuber (g.cuber): 'promotes_inputs' failed to find any matches for the " \
-                   "following pattern: '*'."
-        self.assertEqual(err_msg, expected)
+        p.setup()
```

Comment:
The comment for this test says it will fail, so either update the comment or maybe even get rid of this test.

Reply:
Good catch. Fixed